MdePkg/Library/UefiUsbLib/UsbDxeLib.c

```diff
@@ -650,3 +654,512 @@ UsbClearEndpointHalt (
 
   return Result;
 }
+
+/**
+  Global library data initialization.
+
+  Library public functions' input is the instance of UsbIo protocol. Check if the global
+  data relevant to the UsbIo. If not, read the device and update the global data.
+
+  @param  UsbIo           The instance of EFI_USB_IO_PROTOCOL.
+
+  @retval EFI_SUCCESS     The global data is updated.
+  @retval EFI_NOT_FOUND   The UsbIo configuration was not found.
+
+**/
+EFI_STATUS

```

Comment:
Should make this set of internal helper services `static` so only the APIs added to the lib class are public.

Reply:
done. Note that different libraries in MdePkg have either `static` or `STATIC`, case is aligned with the rest of the declaration. I kept `static`, please see if it is acceptable.

Reply:
lower case `static` is the correct style